docs/developers/documentation.rst

```diff
@@ -68,3 +71,53 @@ Add documentation to the API reference:
     :members:
 
 - Include the file you want to import in the `__init__.py` file properly
+
+Developer center
+----------------
+
+Guides, Examples and Support pages are generated in the developer center from a specific documentation branch.
+Each release has a `docs/v.M.m.p` reference branch, that is generated at the same time the release tag is created. The difference is that
+we have separated branches in order to be able to change the docs after the release.
+
+Writing Documentation
+~~~~~~~~~~~~~~~~~~~~~
+
+Documentation is placed in `/docs/developer-center` directory.
+
+- In order to test it in the developer center, it's necessary to change the branch to be tested in the `config.js` (instructions are explained in the developer center repository)
+- In order to modify the documentation of a specific version, just:
+    
+Download the docs branch and create from it a new one:
+
+.. code::
+    $ git fetch origin docs/v.M.m.p
+    $ git checkout docs/v.M.m.p
+    $ git checkout -b docs/v.M.m.p-your-fix-description
```

Comment:
We should try to avoid creating PRs with the `docs/*` because we can not remove these branches.

Reply:
OUCH!! veeeery good catch!! let me change that